DevDeployment/src/main.tf

```diff
@@ -38,7 +51,7 @@ module "mock_webapp_service" {
     "WEBSITE_ENABLE_SYNC_UPDATE_SITE"                      = "true"
     "APPINSIGHTS_INSTRUMENTATIONKEY"                       = "NOT_CONFIGURED"
   }
-  tags = local.tags
+  tags                      = local.tags
```

Comment:
I think you're trying to line the = sign up with the others, but instead doesn't seem to line up with anything.

Reply:
Hi,
Fixed the indentation.